.travis.install.sh

```diff
@@ -5,7 +5,7 @@ if [ "$TRAVIS_PHP_VERSION" = 'hhvm' ] || [ "$TRAVIS_PHP_VERSION" = 'hhvm-nightly
     hhvm composer-installer.php
     hhvm -v ResourceLimit.SocketDefaultTimeout=30 -v Http.SlowQueryThreshold=30000 composer.phar update --prefer-source
 elif [ "$TRAVIS_PHP_VERSION" = '5.3.3' ] ; then
-    composer self-update
+    ## Can't update on php 5.3.3 as openssl is now required in the latest build and not available on php5.3.3 at travis
```

Comment:
Can do... Just needs a minor version bump
On Apr 2, 2016 22:42, "mikeSimonson" <notifications@github.com> wrote:

> In .travis.install.sh
> <https://github.com/doctrine/instantiator/pull/25#discussion_r58299598>:
>
> > @@ -5,7 +5,7 @@ if [ "$TRAVIS_PHP_VERSION" = 'hhvm' ] || [ "$TRAVIS_PHP_VERSION" = 'hhvm-nightly
> >      hhvm composer-installer.php
> >      hhvm -v ResourceLimit.SocketDefaultTimeout=30 -v Http.SlowQueryThreshold=30000 composer.phar update --prefer-source
> >  elif [ "$TRAVIS_PHP_VERSION" = '5.3.3' ] ; then
> > -    composer self-update
> > +    ## Can't update on php 5.3.3 as openssl is now required in the latest build and not available on php5.3.3 at travis
>
> Good for me. Should I drop 5.4 too ? Le sam. 2 avr. 2016 12:17, Marco
> Pivetta <notifications@github.com> a écrit :
> … <#m_-8486183839239942810_>
> In .travis.install.sh <#25 (comment)
> <https://github.com/doctrine/instantiator/pull/25#discussion_r58291482>>:
> > @@ -5,7 +5,7 @@ if [ "$TRAVIS_PHP_VERSION" = 'hhvm' ] || [
> "$TRAVIS_PHP_VERSION" = 'hhvm-nightly > hhvm composer-installer.php > hhvm
> -v ResourceLimit.SocketDefaultTimeout=30 -v Http.SlowQueryThreshold=30000
> composer.phar update --prefer-source > elif [ "$TRAVIS_PHP_VERSION" =
> '5.3.3' ] ; then > - composer self-update > + ## Can't update on php 5.3.3
> as openssl is now required in the latest build and not available on
> php5.3.3 at travis We can just drop PHP 5.3 compat. — You are receiving
> this because you modified the open/close state. Reply to this email
> directly or view it on GitHub <
> https://github.com/doctrine/instantiator/pull/25/files/d5dca2930620b618c5454636b18ab103856e78de#r58291482>
>
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/doctrine/instantiator/pull/25/files/d5dca2930620b618c5454636b18ab103856e78de#r58299598>
>